### DISCUSSION TITLE
Products created in the backend admin are not showing up on the frontend

### DISCUSSION THREAD

**Discussion by jackomcpie** [solidusio/solidus#5066] [Troubleshooting]
Hello, I've created a few products in the backend admin page however, it is not showing up in the front end page. Neither are the sample ones given by Solidus. It says no products found. Anyone knows how to fix this please? Thank you in advance.

![image](https://user-images.githubusercontent.com/132438276/236116695-1f6cff72-51fc-45f4-9630-520140525a85.png)
![image](https://user-images.githubusercontent.com/132438276/236116781-8b90c7b0-f145-446e-b4a8-7934043c2ee6.png)

**Comment by jarednorman** [comment]
Is the available date set?

**Comment by Moriort** [comment]
i have the same problem

**Reply by jarednorman** [reply]
Have you made the product available, i.e. set the available date?
